### PR TITLE
Cart and wishlist added

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Look into for changes
https://vibe-store-git-cart-bhargavirdy.vercel.app/wishlist.html
https://vibe-store-git-cart-bhargavirdy.vercel.app/cart.html for changes
image
image

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        <div class="cart-items">
+            <div class="cart-item card card-h flex flex-row py-2 px-4">
+                <img class="ml-3" src="./images/watch1.jpeg" alt="kids-dress" />
+                <div class="card-content flex flex-column ml-4">
+                    <span class="h6">Casual Watch</span>
+                    <span class="">Boys Daily wear for neog camp</span>
+                    <span class="text-regular-weight mt-2 mb-0">
+                        <span class="h5">Rs.899</span>
+                        <span class="text-secondary"><span class="text-strike-through">Rs.999</span><span
+                                class="text-primary">10%</span></span>
```
Comment (mariyasada) [inline cart.html:53]: add off keyword like 10% off

**Review by mariyasada — APPROVED** [review]
it looks nice

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        </div>
+
+        <div class="navbar--search">
+            <span class="searchbar-btn"><i class="fa fa-search"></i></span>
+            <input class="searchbar-input" type="text" name="search" id="navbar-search" placeholder="Search" />
+        </div>
+
+        <ul class="ul-none navbar--icons">
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-user"></i></a>
```
Comment (Kothalkarpriya) [inline cart.html]: use dead link as# if you don't have anything to put their as of now

File: css/cart.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+:root{
+    --heart-color: #d13d23;
+}
+* {
+	margin: 0;
+	font-family: 'Lato', sans-serif;
```
Comment (Kothalkarpriya) [inline css/cart.css]: use code formatter

File: wishlist.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        </div>
+
+        <div class="navbar--search">
+            <span class="searchbar-btn"><i class="fa fa-search"></i></span>
+            <input class="searchbar-input" type="text" name="search" id="navbar-search" placeholder="Search" />
+        </div>
+
+        <ul class="ul-none navbar--icons">
+            <li class="list-inline-item navbar--icon px-2">
+                <a href="" class="btn"><i class="fas fa-user"></i></a>
```
Comment (Kothalkarpriya) [inline wishlist.html]: here also use dead link

File: wishlist.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+    <div class="wishlist-container">
+        <div class="card card-v flex flex-column py-2 px-2">
+            <img class="badge-container" src="../images/watch1.jpeg" alt="kids-dress" />
+            <span class="badge badge-text bg-primary px-2 py-1">New</span>
+            <div class="card-content flex flex-column mx-2">
+                <span class="h6">Kids Uniform
+                    <span class="heart float-right"> <i class="fas fa-heart"></i></span>
+                </span>
+                <span class="">Boys Daily wear for neog camp</span>
```
Comment (Kothalkarpriya) [inline wishlist.html]: THeir is no class name included then remove the class =""

File: wishlist.html [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+            </div>
+        </div>
+        <div class="card card-v flex flex-column py-2 px-2">
+            <img class="badge-container" src="../images/watch1.jpeg" alt="kids-dress" />
+            <span class="badge badge-text bg-primary px-2 py-1">New</span>
+            <div class="card-content flex flex-column mx-2">
+                <span class="h6">Kids Uniform
+                    <span class="heart float-right"> <i class="fas fa-heart"></i></span>
+                </span>
+                <span class="">Boys Daily wear for neog camp</span>
```
Comment (Kothalkarpriya) [inline wishlist.html]: here also

File: wishlist.html [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
```
Comment (Kothalkarpriya) [inline wishlist.html]: here also

File: wishlist.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
```
Comment (Kothalkarpriya) [inline wishlist.html]: here also

File: wishlist.html [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
```
Comment (Kothalkarpriya) [inline wishlist.html]: here also

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: wishlist.html (hunk 9 above)
Comment (bhargaviRdy) [reply wishlist.html]: made all the changes

**Pull request merged**